examples/pytorch/graphsage_base/node_classification.py

```diff
@@ -0,0 +1,177 @@
+import torch

```

Comment:
It's a bit difficult to understand what changes have been done against the existing example script. Could you directly modify or replace that one so github can generate a comparison here?

Reply:
Sure, let me copy graphsage_base/node_classification.py to replace the old graphsage/node_classification.py. If we directly modify the old one instead of creating a new folder, I am afraid we're going to forget which file is the golden one again after couple months looking back.

Reply:
Ultimately, the scripts under `graphsage` should all be standardized so I will not worry about it.